libs/external-router/driver/magento/src/model/magento-url-resolver.ts

```diff
@@ -0,0 +1,5 @@
+import { MagentoResolution } from './resolution';
+
+export interface MagentoUrlResolver {
+  urlResolver: MagentoResolution;
```

Comment:
I would expect this to be named similarly to the model it is associated with. Like "urlResolution"